.goreleaser.yaml

```
@@ -180,14 +184,183 @@ dockers:
     - "--build-arg=BUILD_ENV=goreleaser"
     - "--platform=linux/arm64/v8"
 
+  # fleet-controller images
```

nit: we could perhaps differentiate comments here, such as:
```suggestion
  # Prime fleet-controller images
```